broker/src/main/java/io/zeebe/broker/exporter/stream/ExporterDirector.java

```diff
@@ -129,7 +132,7 @@ protected void onActorStarted() {
       eventFilter = createEventFilter(containers);
       LOG.debug("Set event filter for exporters: {}", eventFilter);
 
-    } catch (final Throwable e) {
```

Comment:
If you remember why we're catching throwable and not exception then I'll gladly revert, otherwise this is just strange to me.